src/main/java/bg/libapp/libraryapp/model/mappers/AuthorMapper.java

```diff
@@ -0,0 +1,51 @@
+package bg.libapp.libraryapp.model.mappers;
+
+import bg.libapp.libraryapp.model.dto.author.AuthorBookViewDTO;
+import bg.libapp.libraryapp.model.dto.author.AuthorViewDTO;
+import bg.libapp.libraryapp.model.entity.Author;
+import bg.libapp.libraryapp.repository.AuthorRepository;
+import bg.libapp.libraryapp.repository.BookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+import java.util.stream.Collectors;
+
+@Component
+public class AuthorMapper {
```

Comment:
I think it is a good idea for all the Mapper classes to be static.

Reply:
Actually I started with this idea, but there was a problem because the mapper communicated with the repositories.
However @AtanasAtanasovv commented that mappers shouldn't communicate with the data layer and I refactored the mappers.
As soon as I finish with task#3 I'll try to make the mappers again static.